Resources/AzureDevOps/VariableTemplates/LabEnvironmentVariables.yml

```diff
@@ -2,4 +2,6 @@ variables:
 - name: ReleaseEnvironmentName
   value: LAB
 - name: FunctionAppName
-  value: dfc-lab-fam-arearoute-fa
\ No newline at end of file
+  value: dfc-lab-fam-arearoute-fa
+- name: sharedKeyVaultName
```

Comment:
FYI,  `sharedKeyVaultName` is actually provided via the library group `dfc-shared-<env>`  which is included int the pipeline.

I'm actually thinking having it in the yaml variable templates is better, so lets keep it!